### PR TITLE
Tweaks some storage issues with the explorer belt.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
The bulky pickaxes were making them unable to store the amount of shit I wanted them to. So this should make them better and as they were intended.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -127,6 +127,7 @@
 	storage_slots = 5
 	w_class = 4
 	max_w_class = 4 //Pickaxes are big.
+	max_combined_w_class = 20 //Not an issue with this whitelist, probably.
```
Comment (optimumtact) [inline code/game/objects/items/weapons/storage/belt.dm]: DODGY AS FUCK

Comment (TechnoAlchemisto) [inline code/game/objects/items/weapons/storage/belt.dm]: Do not worry my friend haha bro it's balanced

Comment (octareenroon91) [inline code/game/objects/items/weapons/storage/belt.dm]: IT'S JUST FOUR PICKS

Comment (TechnoAlchemisto) [inline code/game/objects/items/weapons/storage/belt.dm]: gotta mine hard

**Comment by KorPhaeron** [comment]
Pickaxes will never be good

**Pull request merged**